### PR TITLE
Clarify argument types in datetime docs.

### PR DESCRIPTION
**Pull request by jugmac00**
"Arguments may be integers... " could be misunderstand as they also
could be strings.

New wording makes it clear that arguments have to be integers.

modified:   Doc/library/datetime.rst

Automerge-Triggered-By: @pganssle

### CODE REVIEW

**Comment by jugmac00** [comment]
Also see #15008

**Review by brandtbucher — APPROVED** [review]
This can probably `skip issue` and `skip news`. Do you mind changing the title to be more descriptive, though? Maybe something like "Clarify argument types in datetime docs."

**Comment by jugmac00** [comment]
@brandtbucher Thank you for the review. Changed the title as you required.

**Comment by brandtbucher** [comment]
@jugmac00 Sorry, I actually meant the GitHub PR title. The commit message is fine!

**Comment by jugmac00** [comment]
Updated the title of the PR.

I have not touched the commit message again - if you want me to switch back to "Improve wording." title - just give a quick note, otherwise I hope the PR is fine now :-)

Thanks for your time and effort!

**Review by ned-deily — APPROVED** [review]
LGTM.  Awaiting review by one of the datetime core developers (@abalkin or @pganssle).

**Review by pganssle — APPROVED** [review]
LGTM, thanks @jugmac00.

Sorry I accidentally closed this PR, I actually have no idea how that happened.

**Pull request merged**
